lib/route.js

```diff
@@ -307,12 +307,28 @@ function buildRouting (options) {
         constraints.version = opts.version
       }
 
-      const headHandler = router.find('HEAD', opts.url, constraints)
-      const hasHEADHandler = headHandler != null
+      let headHandler = null
+      let hasHEADHandler = false
+
+      if (shouldExposeHead && opts.method === 'GET') {
+        // We're using the "router.on" method in order to understand if there is an identical route
```

Comment:
Can we add a note to indicate that this will prevail until `find-my-way` implements a `hasRoute` or similar feature?

Reply:
What do you mean by note? I wrote in the comment below: "Once 'has' method will be implemented on 'find-my-way' side, we will use it"

Reply:
Ah sorry, I overlooked. nvm 👍